stock_picking_adj/views/stock_picking_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='force_assign']" position="replace"/>
```

Comment:
Can we make the field invisible instead?  Want to avoid "replace" where we can as it sometimes breaks inheritance.